requirements/pip.txt

```diff
@@ -2,7 +2,7 @@
 pip==8.1.1
 virtualenv==15.0.1
 docutils==0.11
-Sphinx==1.3.5
+Sphinx==1.5.3
```

Comment:
I'm not sure if we still even need these as requirements for the code, since all the Sphinx interaction happens inside Docker. I guess for non-docker installs it's still required, so might as well update it.